lib/modules/datasource/github-runners/index.ts

```diff
@@ -9,17 +9,25 @@ export class GithubRunnersDatasource extends Datasource {
    * Only add stable runners to the datasource. See datasource readme for details.
    */
   private static readonly releases: Record<string, Release[] | undefined> = {
-    ubuntu: [{ version: '22.04' }, { version: '20.04' }, { version: '18.04' }],
+    ubuntu: [
+      { version: '22.04' },
+      { version: '20.04' },
+      { version: '18.04', isDeprecated: true },
```

Comment:
```suggestion
      { version: '18.04', isDeprecated: true },
      { version: '16.04', isDeprecated: true },
```

Now we're deprecating old stuff, should we add and deprecate `16.04` as well?

- https://github.com/actions/runner-images/pull/8857